bootstrapvz/providers/gce/tasks/image.py

```diff
@@ -56,6 +26,26 @@ def run(cls, info):
 		if 'description' in info.manifest.image:
 			image_description = info.manifest.image['description']
 		log_check_call(['gcutil', '--project=' + info.manifest.image['gce_project'],
-		                'addimage', info._gce['image_name'],
-		                info.manifest.image['gcs_destination'] + info._gce['tarball_name'],
+		                'addimage', info.image['name'],
+		                info.manifest.image['gcs_destination'] + info.image['tarball_name'],
 		                '--description=' + image_description])
+
+
+class GenerateImageTarballName(Task):
+	description = 'Generating custom image tarball name'
+	phase = phases.image_registration
+	successors = [image.CreateImageTarball]
+
+	@classmethod
+	def run(cls, info):
+		import datetime
+		today = datetime.datetime.today()
+		name_suffix = today.strftime('%Y%m%d')
+		image_name_format = '{lsb_distribution}-{lsb_release}-{release}-v{name_suffix}'
+		image_name = image_name_format.format(lsb_distribution=info._gce['lsb_distribution'],
```

Comment:
I've read about [Debian dropping the Linux Standard Base][1] yesterday, but I'm not sure what we could do here. The **only** LSB information used here by GCE that isn't available at the manifest is the `major.minor` release number, e.g.: `debian-8-2-jessie-v20151017.tar.gz` (`8-2` is the replaced `8.2`). We can get the major version from [our `releases` module][2], but what about the minor?

[1]: https://lwn.net/Articles/658809/
[2]: https://github.com/andsens/bootstrap-vz/blob/0068cdc1a8a15061856a962045984126476b9583/bootstrapvz/common/releases.py#L34